### DISCUSSION TITLE
Forward/back between files (assumed using Ctrl + I, Ctrl + O)

### DISCUSSION THREAD

**Discussion by LyndonGingerich** [JetBrains/ideavim#641] [Ideas]
I'm trying Vim for the first time, so if I'm open to finding out that I am missing something.

The forward/back buttons on my mouse can navigate move between Rider tabs; I use them heavily. Ctrl + I and Ctrl + O from IdeaVim can't. I don't want to have to grab for my mouse frequently. Is the difference in behavior intended?

**Comment by raycon** [comment]
I use the following mapping to solve this problem.
```
nmap <C-o>      <Action>(Back)
nmap <C-i>      <Action>(Forward)
```

**Comment by ManuelTS** [comment]
You can move in additional ways (or rather commands) between buffers, see my post here:
https://github.com/JetBrains/ideavim/discussions/287

**Comment by gavinchou** [comment]
Making vim shortcuts overriding idea defaults in vim plugin settings works for me.
<img width="2639" height="1884" alt="image" src="https://github.com/user-attachments/assets/7ce61ddd-0e27-4953-8952-474a4212a37a" />

**Reply by huyz** [reply]
Yep, nowadays, it seems that's all you need to do. no need to create new mappings.
